src/Clustering/GMUClusterManager.h

```diff
@@ -45,13 +45,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface GMUClusterManager : NSObject<GMSMapViewDelegate>
 
+/** Disallow default initializer */
+- (id)init __attribute__((unavailable("Use initWithMap:algorithm:renderer: instead.")));
```

Comment:
I followed latest Apple API's, making it possible to print a message together with the error, so the dev will know which initializer to use, see [here](http://nshipster.com/__attribute__/). `NS_UNAVAILABLE` is fine, but not really helpful for the dev-workflow. You decide!